pom.xml

```diff
@@ -1205,6 +1210,7 @@
 				<module>aws</module>
 				<module>cmis</module>
 				<module>aspose</module>
+				<module>dbms</module>
```

Comment:
Ik denk dat deze module bij de `default-modules` hierboven moet, ipv. de `extra-modules`. In ieder geval zolang `core` nog de JDBC listeners en senders bevat en dus afhankelijk is van de `dbms` bevat.

Reply:
Ah dat is idd logisch, maakt de volgorde daar nog uit, of gewoon onderaan zetten?